app/models/vacancy.rb

```diff
@@ -137,6 +138,16 @@ def organisation
     organisations.find(&:trust?) || publisher_organisation || organisations.first&.school_groups&.first
   end
 
+  def schools
```

Comment:
This feels like its in the wrong place. It's called 'schools' but it doesn't return schools which feels very confusing for a model method.